--- a/Client/Client.csproj
+++ b/Client/Client.csproj
@@ -32,7 +32,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="FFmpeg.AutoGen" Version="5.0.0" />
+    <PackageReference Include="FFmpeg.AutoGen" Version="4.4.1.1" />
     <PackageReference Include="LibUsbDotNet" Version="3.0.97-alpha" />
     <PackageReference Include="SysDVR.Client.Native" Version="1.0.2" />
   </ItemGroup>
